.travis.yml

```diff
@@ -1,37 +1,40 @@
-language: ruby
+language: minimal
 dist: bionic
-sudo: required
 branches:
   only:
   - master
 
-rvm:
-- 2.6.6
-before_script:
-  - gem install bundler -v 1.16.2
-
 script:
-  - make test STACK="cedar-14" TEST_CMD="${TESTFOLDER}"
-  - make test STACK="heroku-16" TEST_CMD="${TESTFOLDER}"
-  - make test STACK="heroku-18" TEST_CMD="${TESTFOLDER}"
+  - make test STACK="${STACK}" TEST_CMD="${TEST_CMD}"
 
 jobs:
   include:
-  - stage: Bash linting (shellcheck)
-    sudo: false
+  - name: Bash linting (shellcheck)
     script: make check
-  - stage: Hatchet Integration
+  - name: Hatchet integration tests
     if: env(TRAVIS_PULL_REQUEST_SLUG) = env(TRAVIS_REPO_SLUG)
-    name: Run Hatchet
+    language: ruby
+    rvm:
+      - 2.6.6
+    before_script:
+      - gem install bundler -v 1.16.2
```

Comment:
Is there a reason for bundler 1.16.x? The last 1.x is 1.17.3

Reply:
Yeah I wondered that too - I left as is since I was planning on updating all the hatchet-related dependencies at some point, and this version is pre-existing.

Reply:
Updated in #1036